travis/before_install.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Don't source setup.sh here, because the virtualenv might not be set up yet
+
+# Install dependencies
+# removing the old c++ link is needed because otherwise brew install fails
+rm '/usr/local/include/c++'
+brew install ccache protobuf gcc@5
```

Comment:
Why is gcc5 needed? I think on OSX people normally use clang.

Reply:
fixed